src/library/playlisttablemodel.cpp

```diff
@@ -327,6 +328,20 @@ void PlaylistTableModel::shuffleTracks(const QModelIndexList& shuffle, const QMo
     m_pTrackCollectionManager->internalCollection()->getPlaylistDAO().shuffleTracks(m_iPlaylistId, positions, allIds);
 }
 
+const QList<int> PlaylistTableModel::getSelectedPositions(const QModelIndexList& indices) const {
+    if (indices.isEmpty()) {
+        return {};
+    }
+    QList<int> positions;
+    // TODO Transpose m_trackPosToRow ?? Would it be faster?
```

Comment:
This would be slightly faster here but comes with a penalty on every select call so probably not worth the effort IMHO. Remove the comment?